### ISSUE TITLE
Support multiple TypeLabels for a child

### ISSUE THREAD

**Issue by victor5171** [ZacSweers/MoshiX#61]
Unfortunately, there's a use-case on my project that needs a specific child of a sealed class to be converted supporting 2 different types. 

I tried to repeat the @TypeLabel with different values, but this not supported yet. Would it be possible to support this use-case?

Thanks!

**Comment by ZacSweers** [comment]
Can you unpack that a bit for me, maybe with an example? Is the ask to be able to do this?

```kotlin
@TypeLabel("orange")
@TypeLabel("clementine")
class Orange(val size: Int) : Fruit()
```

**Comment by ZacSweers** [comment]
II took a crack at what I think you're describing in #62. Let me know what you think

**Comment by victor5171** [comment]
Hello Zac! Sorry for not being clear enough :)

It's exactly what was thinking, something like @TypeLabels(types=["orange", "clementine"])

**Comment by ZacSweers** [comment]
Cool. I don't want to add another annotation but the addition of an extra array field works well enough

**Comment by ZacSweers** [comment]
I've released 0.6.0 with this functionality now. Thanks for the suggestion!

**Comment by victor5171** [comment]
Hey, sorry for the late reply. I updated the library and it works well!

Thanks for the update!
